external/fv3fit/fv3fit/emulation/transforms/transforms.py

```diff
@@ -71,8 +71,37 @@ def backward(self, x: tf.Tensor) -> tf.Tensor:
         return tf.math.exp(x)
 
 
+@dataclasses.dataclass
+class PositiveTransform:
+    """
+    A univariate transformation for::
+
+    y := x
+    x := ReLU(y)
+
+    Attributes:
+        enforce_positive: used only for identifying this transform in the
+            configuration
+    """
+
+    # TODO: Is there a better way to do config instance via dacite w/o uniqueness?
+    enforce_positive: str = "yes_please"
+
+    def __post_init__(self):
+        self.relu = tf.keras.layers.ReLU()
+
+    def forward(self, x: tf.Tensor) -> tf.Tensor:
+        return x
+
+    def backward(self, x: tf.Tensor) -> tf.Tensor:
+        return self.relu(x)
```

Comment:
Can use a function instead of the RelU layer since it is has no tunable parameters, best to avoid `__post_init__` if we can IMO.
 
```suggestion
    def forward(self, x: tf.Tensor) -> tf.Tensor:
        return x

    def backward(self, x: tf.Tensor) -> tf.Tensor:
        return tf.keras.activations.relu((x)
```